### PR TITLE
delete linux from fail port from tomlplusplus

### PR DESCRIPTION
**Pull request by marshevms**
- What does your PR fix?
Build on GNU/Linux
If you can confirm success build on other platforms I also can delete them from cmake file

### CODE REVIEW

File: ports/tomlplusplus/CONTROL [hunk 1]
```diff
@@ -2,4 +2,4 @@ Source: tomlplusplus
 Version: 2.3.0
```
Comment (NancyLi1013) [inline ports/tomlplusplus/CONTROL:2]:
```suggestion
Version: 2.3.0
Port-Version: 1
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM now, thanks for our PR. @marshevms.

**Comment by traversaro** [comment]
I think the other platforms were disabled mostly due to the right version of meson not available in the CI machine. If anyone is interested, probably just enabling them and check if CI is able to run correctly I guess is probably enough.

**Comment by NancyLi1013** [comment]
@traversaro 

Thanks for your kindly reminder. We will check this later.

**Comment by NancyLi1013** [comment]
Confirmed `tomlplusplus `can work on osx platform now. I can build it without any problems on my local.

**Pull request merged**